README.md

```diff
@@ -4,130 +4,180 @@
+out the following documentation.
+
+> Beware: the project is moving very quickly, and thus some of these documents
+> are slightly out of date as compared to the current state-of-the-art.
+
 - [Automatic Differentiation Whitepaper](docs/AutomaticDifferentiation.md)
 - [Automatic Differentiation Manifesto](https://gist.github.com/rxwei/30ba75ce092ab3b0dce4bde1fc2c9f1d)
 - [Python Interoperability](docs/PythonInteroperability.md)
 - [Aggregate Parameters and Parameter Update](docs/AggregateParameters.md)
-
-### Swift API reference
-
-- [TensorFlow](https://www.tensorflow.org/api_docs/swift/Structs/Tensor)
-
-### Design proposals
-
-- [One Graph Function per Host Function](proposals/OneGraphFunctionPerHostFunction.md)
+- [Graph Program Extraction](docs/GraphProgramExtraction.md)
 
 ## Source code
 
-The active development of Swift for TensorFlow will happen under the 
-"tensorflow" branch of
-[apple/swift](https://github.com/apple/swift/tree/tensorflow).
-
-These projects include:
+Compiler and standard library development of Swift for TensorFlow happens
+on the `tensorflow` branch of the
+[apple/swift](https://github.com/apple/swift/tree/tensorflow) repository.
 
-- The compiler and standard libraries: [apple/swift](http://github.com/apple/swift/tree/tensorflow)
-- Debugger and REPL support: [apple/swift-lldb](http://github.com/apple/swift-lldb/tree/tensorflow)
+Additional code repositories that make up the core of project include:
 
-Swift for TensorFlow is **not** intended to remain a long-term fork of the official 
-Swift language. New language features will eventually go through the Swift evolution process
-as part of being considered for being pulled into master.
+ - [Debugger and REPL support](http://github.com/apple/swift-lldb/tree/tensorflow)
+ - [High level APIs](https://github.com/tensorflow/swift-apis)
 
-## Models and Tutorials
-
-You can find example machine learning models at
-[tensorflow/swift-models](https://github.com/tensorflow/swift-models), or try out a tutorial at
-[tensorflow/swift-tutorials](https://github.com/tensorflow/swift-tutorials).
-
-## Related Projects
+> Swift for TensorFlow is **not** intended to remain a long-term fork of the official 
+> Swift language. New language features will eventually go through the Swift evolution process
+> as part of being considered for being pulled into master.
 
 ### Jupyter Notebook support
 
-[Jupyter Notebook](http://jupyter.org/) support for Swift for TensorFlow is under development at
+[Jupyter Notebook](http://jupyter.org/) support for Swift is under development at
 [google/swift-jupyter](https://github.com/google/swift-jupyter).
 
 ## Community
 
-Discussion about Swift for TensorFlow happens on the
+Swift for TensorFlow discussions happen on the
 [swift@tensorflow.org mailing list](https://groups.google.com/a/tensorflow.org/d/forum/swift).
 
-## Bugs Reports and Feature Requests
+For the most part, the core team's development is tracked in
+[JIRA].
```

Comment:
Looks like this sentence is a duplicate of what comes in the next section?

Reply:
Great catch @xeno-by ! It's tinkercruft leftover from when I was working through it. (Hence the broken link.) It should now be removed.